app/views/template_invocations/show.html.erb

```diff
@@ -1,4 +1,13 @@
-<% title _('Detail of %s run') % @template_invocation.job_invocation.job_category %>
+<% items = [{ :caption => @template_invocation.job_invocation.description,
+              :url => job_invocation_path(@template_invocation.job_invocation_id) },
+            { :caption => _('Template Invocation for %s') % @template_invocation.host.name }] %>
+
+<% breadcrumbs(:resource_url => template_invocations_api_job_invocation_path(@template_invocation.job_invocation_id),
+               :name_field => 'host_name',
+               :switcher_item_url => '/template_invocations/:id',
```

Comment:
Is it possible to use rails helper to create the `switcher_item_url`?
Example:
https://github.com/theforeman/foreman/blob/833e2aef24d6f3cb6eae90213ea198dd354a5709/app/views/provisioning_templates/edit.html.erb#L2